config/enterprise_versions.yml

```diff
@@ -86,8 +86,8 @@ components:
     image: tigera/elasticsearch-metrics
     version: master
   prometheus:
-    image: prometheus/prometheus
+    image: tigera/prometheus
     version: v2.17.2
   prometheus-alertmanager:
-    image: prometheus/alertmanager
+    image: tigera/alertmanager
     version: v0.20.0
```

Comment:
Looks like enterprise_versions.yml is a bit mangled up (over time and not because of this PR). Noticed that there is a comment about 3rd party images but it has enterprise images in them. Can you 
1. Update the comment about third party images 
2. Reorder the images around the comment so that the comment about 3rd party images is retained but elasticsearch-operator, prometheus and alertmanager are moved under the comment and no other non 3rd party components are under it.

Reply:
https://github.com/tigera/operator/pull/1352/commits/88527adb4dcc3fa84f806620b4b75a42b2324bcd